src/inlineeditorui.js

```diff
@@ -132,6 +136,10 @@ export default class InlineEditorUI extends EditorUI {
 
 		// https://github.com/ckeditor/ckeditor5-editor-inline/issues/4
 		view.listenTo( editor.ui, 'update', () => {
+			if ( !editableElement.ownerDocument.body.contains( editableElement ) ) {
```

Comment:
Why do we need this condition? It wasn't necessary before and I see some tests but there's no comment here.

Reply:
You're right... I forgot to remove it after changes...

Reply:
You're right... I forgot to remove it after changes...